src/routes/index.js

```diff
@@ -1,11 +1,37 @@
 import express from 'express';
+import passport from 'passport';
 
 const router = express.Router();
 
-router.get('/', (req, res) => {
+// router.get('/auth/linkedIn', userController.linkedInAuth);
+
+//and then authenticate as:
+
+router.get(
+  '/auth/linkedin',
+  passport.authenticate('linkedin', { state: 'SOME STATE' }),
+  function(req, res) {
```

Comment:
You might want to change this to arrow function for consistency.

Reply:
fixed